src/platform/licensing/dto/licensing.dto.granted.entitlement.ts

```diff
@@ -0,0 +1,13 @@
+import { LicenseEntitlementType } from '@common/enums/license.entitlement.type';
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType('LicensingGrantedEntitlement')
+export class LicensingGrantedEntitlement {
+  @Field(() => LicenseEntitlementType, {
+    description: 'The entitlement that is granted.',
+  })
+  type!: LicenseEntitlementType;
+
+  @Field(() => Number, { nullable: true })
+  limit!: number;
```

Comment:
this should be optional

Reply:
or it should be nullable false...and I think that is the better update.